be/src/olap/memtable.cpp

```diff
@@ -302,7 +302,7 @@ Status MemTable::_do_flush(int64_t& duration_ns) {
         }
     } else {
         vectorized::Block block = _collect_vskiplist_results();
-        RETURN_NOT_OK(_rowset_writer->add_block(&block));
```

Comment:
add some comment here to explain the multi thread problem. So that other person will notice this problem when he or she want to modify or refactor it in the future.